Package.swift

```diff
@@ -1,11 +1,11 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.9
 
 import PackageDescription
 
 let package = Package(
 	name: "SVGView",
 	platforms: [
-		.macOS(.v11),
+		.macOS(.v14),
```

Comment:
bumping to match our packages, also to get rid of some of the unnecessary methods like `cgPath` on NSBezierPath